Tests/js/models/ez-usermodel.html

```diff
@@ -10,26 +11,27 @@
 <script type="text/javascript" src="./assets/ez-usermodel-tests.js"></script>
 <script>
     var filter = (window.location.search.match(/[?&]filter=([^&]+)/) || [])[1] || 'raw',
-        loaderFilter;
-    if (filter == 'coverage'){
-        loaderFilter = {
+        loaderFilter = (filter !== 'coverage') ? filter : {
```

Comment:
can you please stop doing unrelated changes ? This is completely unrelated to the addition of the `loadDrafts` method (not to mention the very limited added value).